02-microarray/00-intro-to-microarray.Rmd

```diff
@@ -6,3 +6,60 @@ output:
     toc: true
     toc_float: true
 ---
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Introduction to Microarray technology](#introduction-to-microarray-technology)
+  - [Microarray data **strengths**:](#microarray-data-strengths)
+  - [Microarray data **limitations**:](#microarray-data-limitations)
+- [About quantile normalization](#about-quantile-normalization)
+- [More resources on microarray technology:](#more-resources-on-microarray-technology)
+  - [Why doesn't the gene I care about show up in this microarray dataset?](#why-doesnt-the-gene-i-care-about-show-up-in-this-microarray-dataset)
+- [References](#references)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Introduction to Microarray technology
+
+Data analyses are generally not "one size fits all"; this is particularly true between RNA-seq vs microarray data.
+This tutorial has example analyses [organized by technology](../01-getting-started/getting-started.html#about-how-this-tutorial-book-is-structured) so you can follow examples that are more closely tailored to the nature of the data at hand. 
+
```

Comment:
Before talking strengths/weakness, what about a brief mention of how microarray data is generated? That will basically give you a word bank to talk about strengths/weaknesses. A nice open source graphic would go a long way to illustrate the process of sample collection to data matrix.

Reply:
Could even include refine.bio as the penultimate stage, and show a smiling researcher at the final stage with clean data in a hand basket lol too much too much

Reply:
I'll probably add a graphic, put a brief sentence about how the data are generated and then link out to a resource that goes into more detail. (I just assume everyone knows about how microarray works, but you know what they say about assuming...)

Reply:
I suppose if microarray gets a graphic, RNA-seq should get a graphic. 😝